src/main/java/edu/mum/sonet/models/Post.java

```diff
@@ -1,23 +1,26 @@
 package edu.mum.sonet.models;
 
+import java.util.HashSet;
 import java.util.Set;
 
-import javax.persistence.Entity;
-import javax.persistence.JoinColumn;
-import javax.persistence.ManyToOne;
-import javax.persistence.OneToMany;
+import javax.persistence.*;
 
 import com.fasterxml.jackson.annotation.JsonIgnoreProperties;
 
 import lombok.Data;
 import lombok.NoArgsConstructor;
+import lombok.Setter;
 
 @Data
 @NoArgsConstructor
+@Setter()
```

Comment:
why you add setter @Data has setter getter

Reply:
I was testing something and forgot to remove it. Thanks for the catch.